JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -128,20 +135,45 @@ fun ArticleScreen(
         topBar = {
             TopAppBar(
                 title = {
-                    Text(
-                        text = "Published in: ${post.publication?.name}",
-                        style = MaterialTheme.typography.subtitle2,
-                        color = LocalContentColor.current
-                    )
+                    Column(modifier = Modifier.padding(top = 10.dp, bottom = 5.dp)) {
+                        Row(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .wrapContentWidth(align = Alignment.CenterHorizontally)
+                                .padding(end = 50.dp)
+                        ) {
+                            Surface(
+                                color = Color(0xff073042),
```

Comment:
This should come from MaterialTheme, and ditto below

Reply:
Changed, I created two new colors in the Material Theme